scripts/gitlab/lib.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

`/bin/bash` is not a standard path and should never be used!

Reply:
In the context of where these scripts run (a docker image we maintain), it's definitely going to be /bin/bash but I agree with the general case. Will fix :)